ts/features/wallet/onboarding/cobadge/screens/start/CoBadgeChosenBankScreen.tsx

```diff
@@ -24,15 +31,40 @@ const tos_url =
 
 const partecipatingBank_url =
   "https://io.italia.it/cashback/carta-non-abilitata-pagamenti-online";
+
+const abiListLoadingError = (
+  isLoading: boolean,
+  onAbort: () => void,
+  onRetry: () => void
+) => (
+  <LoadingErrorComponent
+    testID={"abiListLoadingError"}
+    isLoading={isLoading}
+    loadingCaption={I18n.t("wallet.onboarding.coBadge.start.loading")}
+    onAbort={onAbort}
+    onRetry={onRetry}
+  />
+);
+
 /**
  * The initial screen of the co-badge workflow (starting with a specific ABI, eg. from BANCOMAT screen)
  * The user can see the selected bank and can start the search for all the co-badge for the specific bank.
  * @constructor
  * @param props
  */
-const CoBadgeChosenBankScreen = (props: Props): React.ReactElement => {
+const CoBadgeChosenBankScreen = (props: Props): React.ReactElement | null => {
   const { showModal, hideModal } = useContext(LightModalContext);
 
+  React.useEffect(() => {
+    fold(
+      props.abiListStatus,
+      () => props.loadAbiList(),
+      () => null,
+      _ => null,
+      _ => props.loadAbiList()
+    );
+  }, []);
```

Comment:
```suggestion
  }, [props.abiListStatus]);
```
Could it be an improvement?

Reply:
I think that this can be an error.

I want to run the useEffect only at the mount of the component.

If I use your suggestion the useEffect will be fired every time the `props.abiListStatus` change value.
This can also lead to infinite loop if for example the `/status/abi.json` return sistematically 500:
- When the user go to the Wallet section the `/status/abi.json` return 500 so `props.abiListStatus` will be `remoteError`
- The user enter in this screen, such that the `props.abiListStatus` is `remoteError` the `props.loadAbiList()` will run. This change the state to `remoteLoading`, the useEffect is called but return none
- The `/status/abi.json` return 500 again. This change the state to `remoteError`, the useEffect is called and the loop restart.

Reply:
I think that this can be an error.

I want to run the useEffect only at the mount of the component.

If I use your suggestion the useEffect will be fired every time the `props.abiListStatus` change value.
This can also lead to infinite loop if for example the `/status/abi.json` return sistematically 500:
- When the user go to the Wallet section the `/status/abi.json` return 500 so `props.abiListStatus` will be `remoteError`
- The user enter in this screen, such that the `props.abiListStatus` is `remoteError` the `props.loadAbiList()` will run. This change the state to `remoteLoading`, the useEffect is called but return none
- The `/status/abi.json` return 500 again. This change the state to `remoteError`, the useEffect is called and the loop restart.

Do you agree?